backend/cache.go

```diff
@@ -152,7 +153,8 @@ func (c *cache) GetToken(ctx context.Context, address string) (*models.Token, er
 
 func (c *cache) GetTotals(ctx context.Context, from, to time.Time, interval time.Duration) ([]*models.TotalBucket, error) {
 	k := key(totalsEP, from, to, interval, "")
-	v, err := c.check(k, func() (interface{}, error) {
+	ttl := time.Duration(math.Max(float64(interval), float64(c.ttl)))
+	v, err := c.check(k, ttl, func() (interface{}, error) {
```

Comment:
this means for daily it will be cached for about a day, which for a lot of that time is fine, but is kinda stale after 12 hours where you might want 'today'... it's easy enough to cut this off at 1 hour or something if this is too stinky